src/pages/editor/ChannelClaimTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { ClaimedChannel, SelectedPublicationChannelType } from '../../types/customerInstitution.types';
+import { PublicationChannelType } from '../../types/registration.types';
+import { ChannelClaimTableRow } from './PublicationChannelTableRow';
+
+interface ChannelClaimTableProps {
+  channelClaimList: ClaimedChannel[];
+  channelType: SelectedPublicationChannelType;
+}
+
+export const ChannelClaimTable = ({ channelClaimList, channelType }: ChannelClaimTableProps) => {
+  const { t } = useTranslation();
+  const claimedChannels = filterClaimedChannels({ channelClaimList, channelType }) ?? [];
+
+  return (
+    <TableContainer>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>{t('common.publisher')}</TableCell>
+            <TableCell>{t('editor.institution.channel_claims.channel_owner')}</TableCell>
+            <TableCell>{t('editor.institution.channel_claims.publishing_access')}</TableCell>
+            <TableCell>{t('editor.institution.channel_claims.editing_access')}</TableCell>
+            <TableCell>{t('editor.institution.channel_claims.category_limitations')}</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {claimedChannels.map((channel, index) => (
+            <ChannelClaimTableRow channel={channel} key={index} />
+          ))}
+        </TableBody>
+      </Table>
+    </TableContainer>
+  );
+};
+
+const filterClaimedChannels = ({ channelClaimList, channelType }: ChannelClaimTableProps) => {
```

Comment:
For komponenter må man bruke objekter for props, men her når det er en hjelpefunksjon syns jeg det blir mer tunglest å wrappe parametrene i et objekt.

```suggestion
const filterClaimedChannels = (channelClaimList: ClaimedChannel[], channelType: SelectedPublicationChannelType) => {
```

Kanskje enda enklere å bare ta det inline, og dermed slippe å passe på type osv i argumentene ved at det bare automatisk blir rett der det er definert